### DISCUSSION TITLE
pTemplate not rendering in PrimeNG 16

### DISCUSSION THREAD

**Discussion by TechnicallyTrue** [primefaces/community#1441] [PrimeNG]
I'm using primeng version 16 for a project, and have noticed that PrimeTemplates don't seem to render properly. The problem occurs even if editing [the demo code](https://primeng.org/menu#basic) on stackblitz, so I don't think it's anything about the way my project is set up.
A simple example:
```
<p-menu [model]="menuItems">
  <ng-template pTemplate="start">
    <h1>Hello, world!</h1>
  </ng-template>
</p-menu>
```
The above code _should_, to my understanding, produce a menu with header text saying "Hello, world!" at the top. This does not happen. The menu renders as normal without the header template. This also occurs if the `ng-template` isn't nested within the `p-menu`.
I have imported the `SharedModule` that contains the `PrimeTemplate` class as well.
Am I doing something wrong here, or is this a bug?

**Comment by cetincakiroglu** [comment]
Hi,

Which version are you using? I've tried to reproduce it with v16.9.6-lts and couldn't replicate the issue https://stackblitz.com/edit/sz7j3g?file=src%2Fapp%2Fdemo%2Fmenu-custom-content-demo.html,src%2Fapp%2Fdemo%2Fmenu-custom-content-demo.ts

Could you please provide a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing due to the inactivity.
